libraries/src/CMS/Router/ApiRouter.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Joomla! Content Management System
+ *
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\CMS\ApiRouter;
+use Joomla\Router\Router;
+
+/**
+ * Joomla! API Router class
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class ApiRouter extends Router
+{
+	/**
+	 * Router instances container.
+	 *
+	 * @var    array
+	 * @since  DEPLOY_VERSION
+	 */
+	protected static $instances = array();
+
+	/**
+	 * Constructor.
+	 *
+	 * @param   array  $maps  An optional array of route maps
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function __construct(array $maps = [])
```

Comment:
You don't need to declare the __constructor just to call `parent::__construct()`